config/env/production.js

```diff
@@ -1,20 +1,20 @@
 "use strict";
 
 module.exports = {
-    env: 'production',
-    databaseUrl: process.env.MONGO_URL || 'mongodb://martinbedouret:Kapsch32065@ds253587.mlab.com:53587/cboard',
-    jwt: {
-        secret: process.env.JWT_SECRET
-    },
-    emailTransport: {
-        from: 'cboard@cboard.io',
-        host: 'smtp.sendgrid.net',
-        port: 465,
-        secure: true,
-        service: 'Sendgrid',
-        auth: {
-            user: 'apikey',
-            pass: process.env.SENDGRID_API_KEY
-        }
+  env: "production",
+  databaseUrl: process.env.MONGO_URL || "mongodb://localhost/cboard-api",
```

Comment:
Is there a need for the local fallback in prod mode? If we need to run locally as prod, then cool. Otherwise, might be able to remove that.

Reply:
production configs should not contain sensitive data on open source repo, so this is basically a dummy file to avoid errors, copied from the development file